test/CMakeLists.txt

```diff
@@ -117,6 +117,7 @@ if(NOT (MIOPEN_TEST_GFX900 OR MIOPEN_TEST_GFX906 OR MIOPEN_TEST_GFX908 OR MIOPEN
             OUTPUT_VARIABLE ROCMINFO_OUTPUT
             RESULT_VARIABLE ROCMINFO_EXIT_STATUS
         )
+        message(STATUS "${ROCMINFO_OUTPUT}")
```

Comment:
@cderb This clutters cmake output. Why this is necessary? If something gets wrong you can manually run rocminfo and inspect its output. In CI logs we have `clinfo` output; we can replace `clinfo` with `rocminfo` there if necessary. We can print rocminfo output at highel verbosity level. Finally we can only do that if something gets wrong.

Variants of resolving this comment:
- Replace `clinfo` with `rocminfo` in Jenkinsfile
- Replace STATUS with VERBOSE (`cmake --log-level=VERBOSE ...` will show it)
- (preferred) Only log rocminfo output if MIOPEN_TEST_GPU_DETECTION_FAILED is TRUE.

Maybe I'll do that myself in my next PR,

Reply:
Resolved by #2490